Back-To-School/Assets/BackToSchool/Scripts/GameManagement/Game.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using Assets.BackToSchool.Scripts.Enemies;
+using Assets.BackToSchool.Scripts.Inputs;
+using Assets.BackToSchool.Scripts.Interfaces.Input;
+using Assets.BackToSchool.Scripts.Player;
+using Assets.BackToSchool.Scripts.Progression;
+using Assets.BackToSchool.Scripts.Stats;
+using Assets.BackToSchool.Scripts.UI;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+namespace Assets.BackToSchool.Scripts.GameManagement
+{
+    public class Game : MonoBehaviour
+    {
+        [SerializeField] private HUDPresenter _hudPresenter;
+        [SerializeField] private GameOverPresenter _gameOverPresenter;
+        [SerializeField] private PausePresenter _pausePresenter;
+
+        [SerializeField] private PlayerController _playerPrefab;
+        [SerializeField] private GameObject _inputManagerPrefab;
+        [SerializeField] private GameObject _enemySpawnerPrefab;
+        [SerializeField] private Camera _mainCamera;
+
+        private EnemySpawner _enemySpawner;
```

Comment:
Nice grouping for objects using empty lines. But I'd suggest to move _saveAction lower to separate block. Also, for better readability you can reorder a bit and regroup manager relative to their responsibilities.